miniapps/nurbs/nurbs_ex1.cpp

```diff
@@ -121,11 +127,50 @@ class Diffusion2Integrator: public BilinearFormIntegrator
 
 };
 
+void RefineNURBSFromFile(std::string ref_file, Mesh *mesh)
```

Comment:
Is this function general enough to go into the main library (nurbs.cpp), or do you think it is too specific to the meshes in this example?

Reply:
I think this is a nice improvement.